runtime/src/commonMain/kotlin/pbandk/ExtendableMessage.kt

```diff
@@ -0,0 +1,24 @@
+package pbandk
+
+@Suppress("UNCHECKED_CAST")
+interface ExtendableMessage : Message {
+    @PbandkInternal
+    val extensionFields: AtomicReference<Map<Int, Any>>
```

Comment:
Adding this causes the `test_messages_proto2.kt` generated file in the conformance to fail.  You can see the details in the failed test. Do you have a suggested solution?

Reply:
The failure indicates the override is invalid because it's marked as `@PbandkInternal` on the interface.

Reply:
Ah, that makes sense I suppose. The generated code is trying to `override` the field without first opting-in to the `@PbandkInternal` annotation. I think what makes the most sense here is to instead annotate `ExtendableMessage.extensionFields` with `@PublicForGeneratedCode` rather than `@PbandkInternal`. Since it does need to be accessible by generated code, as the current compiler error points out.